gae_serve/README.md

```diff
@@ -0,0 +1,134 @@
+
+    ```
+    gsutil mb gs://BUCKET_NAME
+    gsutil cp lr.pkl gs://BUCKET_NAME
+    ```
+
+1. Update `app.yaml`:
+
+    * If you already have at least one GAE service in your GCP project:
+
+        - Replace `PROJECT_ID` with your GCP project's id.
+
+        - Replace `BUCKET_NAME` with the name of the bucket you created on GCS above.
+
+        - Replace `CONFIG_ID` with the configuration ID you got from the service endpoint deployment.
+
+    * If you have not deployed any service to GAE with this GCP project:
+
+        - Replace `service: modelserve` with `service: default` in `app.yaml`.  The first service deployed to GAE must be named `default`.
+
+        - Replace `PROJECT_ID` with your GCP project's id.
+
+        - Replace `BUCKET_NAME` with the name of the bucket you created on GCS above.
+
+        - Replace `CONFIG_ID` with the configuration ID you got from the service endpoint deployment.
+
+        - **Note that you will not be able to delete the `default` service from your project.**
+
+    See the [documentation](https://cloud.google.com/appengine/docs/flexible/python/configuring-your-app-with-app-yaml) for more information about `app.yaml`.
+
+1. Deploy the backend service:
+
+    `gcloud app deploy`
+
+    **This step could take several minutes to complete.**
+
+
+1. If the deployment is successful, you can test it from the command line:
+
+    - Create an API key with the "Create credentials" button on the [Credentials](https://console.cloud.google.com/apis/credentials) page.  Make sure you switch to the correct GCP project first.
+
+    - Now you can test the service endpoint: (Remember to replace `PROJECT_ID` and `API_KEY` with their actual values below.)
+
+        * If you have `service: modelserve` in `app.yaml`:
+
+            `curl -H "Content-Type: application/json" -X POST -d '{"X": [[1, 2], [5, -1], [1, 0]]}' "https://modelserve-dot-PROJECT_ID.appspot.com/predict?key=API_KEY"`
+
+        * If you have `service: default` in `app.yaml`:
+
+            `curl -H "Content-Type: application/json" -X POST -d '{"X": [[1, 2], [5, -1], [1, 0]]}' "https://PROJECT_ID.appspot.com/predict?key=API_KEY"`
+
+        You should get the following response:
+
+        `{"y": [0.6473534912754967, -0.7187842827829021, 0.3882338314071392]}`
+
+        The deployed model `lr.pkl` is a simple [linear regression model](http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html) with 2-dimensional inputs.
+
+
+## Clean up
+
+* If the service was deployed as the `default` service (that is, with `service: default` in `app.yaml`), then the service cannot be deleted from the project.
+
+* If the service was deployed as the `modelserve` service (that is, with `service: modelserve` in `app.yaml`), then you can delete service by running:
+
+    `gcloud app services delete modelserve`
+
+    `gcloud service-management delete PROJECT_ID.appspot.com`
+
+
+## Advanced usage
+
+### Healthcheck
```

Comment:
Not important for this iteration of work, but would like to eventually give users a guide to using updated health checks for model serving in the way you suggested in previous comments.

Reply:
I moved this into just a link because it seems that even with the default "legacy" health check, the model is loaded upon the first (health check) request, and the service isn't considered ready until that process is finished (the flask app would be blocking while loading the model).  So there's no additional work needed to use health check to trigger model loading.